src/components/DetailsRequestedFeatures/DetailsRequestedFeatures.js

```diff
@@ -25,24 +27,20 @@ const DetailsRequestedFeatures = ({ projectName, selectedItem }) => {
           ))}
         </div>
         <div className="item-requested-features__table-body">
-          {selectedItem.specFeatures.map(feature => {
-            const project =
-              feature.indexOf('/') > 0 ? feature.split('/')[0] : ''
-            const featureSet = feature.match(/(?<=\/)(.*?)(?=\.)/)
-              ? feature.match(/(?<=\/)(.*?)(?=\.)/)[0].split(':')[0]
-              : feature.split('.')[0]
-            const tag = feature.match(/(?<=\/)(.*?)(?=\.)/)
-              ? feature.match(/(?<=\/)(.*?)(?=\.)/)[0].split(':')[1]
-              : ''
-            const featureName = feature.match(/(?<=\.)(.*?)(?=as)/)
-              ? feature.match(/(?<=\.)(.*?)(?=as)/)[0]
-              : feature.split('.')[1]
-            const alias = feature.match(/(?<=as)(.*?)(&|$)/)
-              ? feature.match(/(?<=as)(.*?)(&|$)/)[0]
-              : ''
+          {selectedItem.specFeatures.map(featureTemplate => {
+            const {
+              project,
+              featureSet,
+              tag,
+              feature,
```

Comment:
If you didn't want to change the name of the variable in the following code from `featureName` to `feature` you could have done here:
```suggestion
              feature: featureName,
```
as part of the object destructuring.
I'm not saying it's important here or that you should change the code here, but I'm just pointing out that the names of RegExp groups in `parseFeatureTemplate` do not mean that the code using this function must adhere to these names. You can choose to use different variable names like I demonstrated above in case it is needed.